wercker.yml

```diff
@@ -1,16 +1,12 @@
-box: dosomething/ds-docker-php
+box: dosomething/ds-docker-php:build-updates
 
 build:
     steps:
       - script:
-          name: install chromedriver
-          code: |-
-            sudo apt-get update
-            sudo apt-get -y install libxpm4 libxrender1 libgtk2.0-0 libnss3 libgconf-2-4
-            sudo apt-get -y install chromium-browser
-      - script:
           name: start mysql
-          code: sudo service mysql start
+          code: |-
+            chown -R mysql:mysql /var/lib/mysql /var/run/mysqld
```

Comment:
This seems to be necessary to [fix a permission when starting MySQL](https://serverfault.com/a/872576) (which seems to be a consequence of a newer MySQL release being included in the new build). I tried applying the fix in the Dockerfile & that didn't work, so I think this is our best bet.